data/channels.csv

```diff
@@ -13229,8 +13229,8 @@ KTVRDT4.us,KTVR-DT4,,opbmusic,,US,US-OR,La Grande,s/US-OR,eng,,FALSE,,,,,https:/
 KTVSLD1.us,KTVS-LD1,,,,US,US-NM,Albuquerque,s/US-NM,eng,,FALSE,,,,,https://i.imgur.com/ECTfuRA.png
 KTVSLD2.us,KTVS-LD2,,Almavision,,US,US-NM,Albuquerque,s/US-NM,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/en/c/c0/Almavision.PNG
 KTVSLD3.us,KTVS-LD3,,The Grio,,US,US-NM,Albuquerque,s/US-NM,eng,,FALSE,,,,,https://i.imgur.com/GKRbUyX.png
-KTVSport.kw,KTV Sport,,,,KW,,,c/KW,eng;ara,sports,FALSE,,,,https://media.gov.kw/,https://i.imgur.com/CKw9kwZ.png
-KTVSportPlus.kw,KTV Sport Plus,,,,KW,,,c/KW,ara,sports,FALSE,,,,,https://i.imgur.com/h3KYxEp.png
+KTVSport.kw,KTV Sport,,,,KW,,,c/KW,eng;ara,sports,FALSE,,,,https://media.gov.kw/,https://i.imgur.com/R1hGX1d.png
+KTVSportPlus.kw,KTV Sport Plus,,KTV Sport 2,,KW,,,c/KW,ara,sports,FALSE,,,,,https://i.imgur.com/l4oX0gf.png

```

Comment:
Alternative channel names must be specified in the third column (https://github.com/iptv-org/database/blob/master/CONTRIBUTING.md#channels)

```suggestion
KTVSportPlus.kw,KTV Sport Plus,KTV Sport 2,,,KW,,,c/KW,ara,sports,FALSE,,,,,https://i.imgur.com/l4oX0gf.png
```